src/Authentication/LoginHandler.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace Bugcache\Authentication;
+
+use Aerys\Bootable;
+use Aerys\Logger;
+use Aerys\ParsedBody;
+use Aerys\Request;
+use Aerys\Response;
+use Aerys\Server;
+use Aerys\ServerObserver;
+use Aerys\Session;
+use Amp\Promise;
+use Amp\Success;
+use Bugcache\Mustache;
+use Bugcache\SessionKeys;
+use Bugcache\Storage\AuthenticationRepository;
+use Bugcache\Storage\UserRepository;
+use function Aerys\parseBody;
+use function Amp\resolve;
+
```

Comment:
totally agree. Just for basic things like aerys\Response or aerys\request, we should use the short way.

Also, as said, I prefer `use Amp\Mysql as mysql;` (with lowercase)...